packages-next/keystone/src/lib/createExpressServer.ts

```diff
@@ -44,8 +47,7 @@ const addApolloServer = ({
     //       // disabled.
     //       tracing: dev,
     //     }),
-    // FIXME: Support for generic custom apollo configuration
-    // ...apolloConfig,
+    ...apolloConfig,
```

Comment:
As a user, If I was to change some unrelated playground option, I probably wouldn't expect auth to stop working so maybe something like this?
```suggestion
    ...apolloConfig,
    playground: {
      ...apolloConfig?.playground,
      settings: {
        'request.credentials': 'same-origin',
      },
    },
```

Reply:
I'd take that a step even further and deep merge our defaults, so that people can override the `settings` if they have a reason to.

(I say this without reviewing what else could be in there, but it seems like you could provide a different thing in `settings` and it would be overwritten if we wrote the merge like this)

Reply:
Whoops, yep, that's what i meant